src/main/java/org/example/MessageServiceImpl.java

```diff
@@ -3,40 +3,35 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
 import java.time.temporal.ChronoUnit;
 
-public class MessageServiceImpl implements MessageService{
+public class MessageServiceImpl implements MessageService {
 
-  private static final LocalDate baselineDate = LocalDate.parse("2012-08-06",
-      DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+  // format for ISO 8601
+  private static final DateTimeFormatter isoFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  private static final LocalDate curiosityLandingDate = LocalDate.parse("2012-08-06",
```

Comment:
Constants are static and final. Very good!
```suggestion
  private static final LocalDate CURIOSITY_LANDING_DATE = LocalDate.parse("2012-08-06",
```
I think the most common approach in Java is to use `MACRO_CASE` (or `SCREAMING_SNAKE_CASE`) for constants to easily tell them from the variables.

Reply:
```
private static final DateTimeFormatter ISO_FORMATE = DateTimeFormatter.ofPattern("yyyy-MM-dd");
  private static final LocalDate CURIOSITY_LANDING_DATE = LocalDate.parse("2012-08-06", ISO_FORMATE);
```
I followed MACRO_CASE (or SCREAMING_SNAKE_CASE)approach thank you for suggesting.